behat.yml

```
@@ -69,6 +69,14 @@ default:
     DrevOps\BehatScreenshotExtension:
       dir: '%paths.base%/.logs/screenshots'
       purge: false # Change to 'true' (no quotes) to purge screenshots on each run.
+      on_failed: true
+      always_fullscreen: true
+      fullscreen_algorithm: resize # 'stitch' (only if GD ext available) or 'resize'
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)